sites/game-client/misc.js

```diff
@@ -6,7 +6,7 @@ function onResize(event) {
   const canvas = document.querySelector('#mapcanvas');
   // TODO: more detailed tweak
   canvas.height = 32 * 15; // 15 tiles
-  canvas.width = 32 * 9; // 9 tiles
+  canvas.width = 32 * 20; // 20 tiles
```

Comment:
I think we can move `misc.js` into `MainUI`. `misc.js` is designed to do the job of main UI previously.
Maybe create `_onResize` as a private method of `MainUI` and `registerOnResize` to let extension register their callback. `window.addEventListener` in `MainUI`'s constructor.

Reply:
Add TODO to `MainUI` if this is to be resolved in future development.